src/isIsogram.test.js

```diff
@@ -1,9 +1,29 @@
-'use strict';
-
+// eslint-disable-next-line strict
 describe('isIsogram', () => {
   const { isIsogram } = require('./isIsogram');
 
-  it(`should be declared`, () => {
+  it('should be a function', () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should return true for isograms', () => {
```

Comment:
```suggestion
  it('should return true for isograms', () => {
```
according to your test name, it should contain assertion with isograms only but it contains non-isogram examples